examples/medical_text_understanding/medical_text_understanding.py

```diff
@@ -0,0 +1,74 @@
+import sys
+from termcolor import colored
+
+from forte.data.data_pack import DataPack
+from forte.data.readers import PlainTextReader
+from forte.pipeline import Pipeline
+from forte.processors.writers import PackIdJsonPackWriter
+
+from ft.onto.base_ontology import (
+    Token,
+)
+
+from ftx.medical.clinical_ontology import Hyponym, Abbreviation
+from fortex.health.processors.scispacy_processor import (
+    ScispaCyProcessor,
+)
+
+def main(
+    input_path: str,
+    process: str,
+):
+    pl = Pipeline[DataPack]()
+    pl.set_reader(PlainTextReader())
+    if process == "abb":
+        pl.add(
+            ScispaCyProcessor(),
+            {
+                "entry_type": "ft.onto.base_ontology.Document",
+                "model_name": "en_core_sci_sm",
+                "pipe_name": "abbreviation_detector",
+                "prefer_gpu": True,
+                "require_gpu": False,
+                "gpu_id": 0,
+            },
+        )
+    elif process == "hyp":
+        pl.add(
+            ScispaCyProcessor(),
+            {
+                "entry_type": "ft.onto.base_ontology.Document",
+                "model_name": "en_core_sci_sm",
+                "pipe_name": "hyponym_detector",
+                "prefer_gpu": True,
+                "require_gpu": False,
+                "gpu_id": 0,
+            },
+        )
```

Comment:
We should handle the exception of user inputting some value other than these 2, it shouldn't enter the flow.